.travis.yml

```diff
@@ -86,46 +86,46 @@ script:
   - ./animate-bmp
   - ./audio_player
   - ./background-effects
-  - ./calculator 
+  #- ./calculator   # https://travis-ci.org/qPCR4vir/nana/jobs/140238537#L1159
```

Comment:
I have no idea what that means. I can't not debug that, sorry. I have simple reported the problem... maybe you understand what is the problem? 
Here  travis #181 I have excluded all the 20 test that don't pass. The other 35 pass. The idea is that we will see if new changes break what is working now. But the link to the failure are there just in case you or somebody have the time to check what is going on. Maybe the (virtual) X server here don't work very well?